<?xml version="1.0"?>
<robot xmlns:xacro="http://ros.org/wiki/xacro" name="simple_tool">
  <link name="my_tool">
    <inertial>
      <origin xyz="0 0 0" />
      <mass value="1.0" />
      <inertia  ixx="0.0" ixy="0.0"  ixz="0.0"  iyy="0.0"  iyz="0.0"  izz="0.0" />
    </inertial>
    <visual>
      <origin xyz="0.0025 0 -0.01" rpy="0 0 0.4"/>
      <geometry>
        <mesh filename="package://affordance_experiment/models/l_stick.dae" />
      </geometry>
    </visual>
    <collision>
      <origin xyz="0.0025 0 -0.01" rpy="0 0 0.4"/>
      <geometry>
        <mesh filename="package://affordance_experiment/models/l_stick.dae" />
      </geometry>
    </collision>
  </link>
  <gazebo reference="my_tool">
    <material>Gazebo/Blue</material>
  </gazebo>
</robot>